tests/tasks/setup/eks/awscli-mng.yaml

```diff
@@ -77,15 +77,16 @@ spec:
       for i in $(seq 1 $asgs)
       do
         #max number of nodes MNG allows per ASG
-        create_dp $i 1000
+        create_dp $i 1000 c5.large,m5.large,r5.large,t3.large,t3a.large,c5a.large,m5a.large,r5a.large
       done
       remaining_nodes=$((nodes%1000))
       if [[ $remaining_nodes -gt 0 ]]
       then
         echo "The remaining_nodes var is greater than 0."
-        create_dp 0 $remaining_nodes
+        create_dp 0 $remaining_nodes c5.large,m5.large,r5.large,t3.large,t3a.large,c5a.large,m5a.large,r5a.large
       fi
-      
+      # Creating an extra asg with 1 node for prometheus server pod scheduling.
```

Comment:
How do you ensure only Prometheus server (and operator) can run on this node?

Reply:
So by default the cl2 test allots cpu and memory to prometheus based on the number of nodes the cluster has https://github.com/kubernetes/perf-tests/blob/release-1.23/clusterloader2/pkg/prometheus/manifests/prometheus-prometheus.yaml#L19. Instead of having a specific node just to run the prometheus pod, what I have tried to ensure is we have at least 1 node which satisfies the cpu and memory request of the prometheus. Definitely, we can have a more advance logic to reserve one node based on taints going forward, but I feel this more simple currently.  Let me know your thoughts on this ?

Reply:
Prometheus server pod may require a very large node when nodes in the cluster are a lot. Have you seen it failing to be scheduled due to nodes are too small?
Some pods under test may end up be scheduled on this large node and it may some inaccuracy in our test.
This is not a show stopper. But we don't plan to address it now. We should track it (at least a TODO).
@hakuna-matatah Thoughts?

Reply:
@mengqiy yeah initially when i tried it was failing due to small nodes.. i had to explicitly adjust my nodes from node groups to add a big schedulable node to my cluster for prometheus.  Current finding is for 10k cluster 1 m5.4xlarge is enough to schedule a pod for Prometheus..maybe we can have a logic to dynamically add one instance based on the cluster nodes and prometheus cpu and memory pre estimation. But yeah open for thoughts on this. +1 to tracking. I will create an issue for this.

Reply:
Given cl2 prom setup is more robust in requesting the [resources](https://github.com/kubernetes/perf-tests/blob/release-1.23/clusterloader2/pkg/prometheus/manifests/prometheus-prometheus.yaml#L19) we can punt adding taints to future if you think it's requires more work. Please cut a backlog as you see fit.
But I generally agree with David, we should be not add any load-test pods to node because there is a risk of prometheus pods getting evicted or OOM killed/restarted which not only it will cost us loosing metrics data but it might also fail our tests because of it's potential impact on APIServer SLOs measurements.

Reply:
Makes sense. I will create a follow up PR for this to have a specific node for prometheus. Thanks @mengqiy @hakuna-matatah